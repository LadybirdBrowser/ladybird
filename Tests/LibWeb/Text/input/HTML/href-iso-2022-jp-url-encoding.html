<!DOCTYPE html>
<html>
<head>
<meta charset="iso-2022-jp">
<script src="../include.js"></script>
<script>
    test(() => {
        const input = "\u4E05"; // ä¸…
        const a = document.createElement("a");
        a.href = "https://ladybird.org/?" + input;
        println(a.search.substr(1));
    })
</script>
</head>
<body></body>
</html>
