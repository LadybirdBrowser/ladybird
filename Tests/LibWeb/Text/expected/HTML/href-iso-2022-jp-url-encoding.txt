%26%2319973%3B
